discord/commands/options.py

```diff
@@ -23,13 +23,40 @@
 """
 
 import inspect
-from typing import Any, Dict, List, Literal, Optional, Union
+from typing import TYPE_CHECKING, Dict, List, Literal, Optional, Type, Union
 from enum import Enum
 
-from ..abc import GuildChannel
+from ..abc import GuildChannel, Mentionable
 from ..channel import TextChannel, VoiceChannel, StageChannel, CategoryChannel, Thread
 from ..enums import ChannelType, SlashCommandOptionType, Enum as DiscordEnum
 
+if TYPE_CHECKING:
```

Comment:
You need a `from __future__ import annotations` import to use this properly without having to stringize the annotations by hand

Reply:
Right, I missed this after testing.